packages/Main/README.md

```diff
@@ -0,0 +1,131 @@
+![iTowns](https://raw.githubusercontent.com/iTowns/itowns.github.io/master/images/itowns_logo_300x134.png)
```

Comment:
I think we should not copy a part of the parent README here but just explains that is contains all other itowns functionalities and link to the parent README of the repo and to the doc instead.

Reply:
cleaned